.github/workflows/regression_test.yml

```diff
@@ -15,104 +15,5 @@ on:
-    - name: Check out the repository
-      uses: actions/checkout@v4
-      with:
-        submodules: true
-    
-    - name: Install softwares
-      run: ./scripts/install.sh
-
-    - name: Build GUIX
-      run: ./scripts/build.sh
-
-    - name: Test GUIX
-      run: ./scripts/test.sh
-      
-    - name: Publish Test Results
-      uses: EnricoMi/publish-unit-test-result-action@v2.11.0
-      if: always()
-      with:
-        files: |
-          ./test/guix_test/cmake/build/*/*.xml
-          
-    - name: Upload Test Results
-      if: success() || failure()
-      uses: actions/upload-artifact@v3.1.3
-      with:
-        name: test_reports_GUIX
-        path: |
-          ./test/guix_test/cmake/build/*.txt
-          ./test/guix_test/cmake/build/*/Testing/**/*.xml
-          ./test/guix_test/cmake/build/**/regression/output_files/*.bin
-
-    - name: Configure GitHub Pages
-      uses: actions/configure-pages@v3.0.6
-      
-    - name: Upload GitHub Pages artifact
-      uses: actions/upload-pages-artifact@v2.0.0
-      with:
-        path: ./test/guix_test/cmake/coverage_report/default_build_coverage
-
-    - name: Generate Code Coverage Summary
-      uses: irongut/CodeCoverageSummary@v1.3.0
-      with:
-        filename: ./test/guix_test/cmake/coverage_report/default_build_coverage.xml
-        format: markdown
-        badge: true
-        hide_complexity: true
-        output: file
-        
-    - name: Write Code Coverage Summary
-      run: |
-        echo "## Coverage Report" >> $GITHUB_STEP_SUMMARY
-        cat code-coverage-results.md >> $GITHUB_STEP_SUMMARY
-
-    - name: Create CheckRun for Code Coverage
-      if: (github.event_name == 'push') || (github.event.pull_request.head.repo.full_name == github.repository)
-      uses: LouisBrunner/checks-action@v1.6.2
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-        name: Code Coverage
-        conclusion: ${{ job.status }}
-        output: |
-          {"summary":"Coverage Report"}
-        output_text_description_file: code-coverage-results.md
-        
-    - name: Add Code Coverage PR Comment
-      if: (github.event_name == 'push') || (github.event.pull_request.head.repo.full_name == github.repository)
-      uses: marocchino/sticky-pull-request-comment@v2
-      with:
-        path: code-coverage-results.md
-
-  deploy_code_coverage:
-    runs-on: ubuntu-latest
-    if: github.event_name == 'push'
-    needs: run_tests
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
-    permissions:
-      pages: write
-      id-token: write
-
-    steps:        
-    - name: Deploy GitHub Pages site
-      id: deployment
-      uses: actions/deploy-pages@v1.2.9
-
-    - name: Write Code Coverage Report URL
-      run: |
-        echo '[Open Coverage Report](${{ steps.deployment.outputs.page_url }})' >>  $GITHUB_STEP_SUMMARY
+    uses: azure-rtos/threadx/.github/workflows/regression_template.yml@master
         
```

Comment:
Need to specify cmake path as this: https://github.com/azure-rtos/threadx/blob/11a7db22b4626a4afd183891b2a8f4fa60a137b4/.github/workflows/regression_test.yml#L19C9-L19C34

Reply:
Fixed.